---
string[] sources
StringArray[] ids
StringArray[] human_messages
StringArray[] data
bool status
string message
